ci/docker/base.ml.Dockerfile

```diff
@@ -1,6 +1,11 @@
 ARG DOCKER_IMAGE_BASE_TEST
 FROM $DOCKER_IMAGE_BASE_TEST
 
+ENV RAY_INSTALL_JAVA=1
+
+RUN apt-get install -y -qq \
+    maven openjdk-8-jre openjdk-8-jdk
```

Comment:
Instead of doing this in the base Dockerfile, can we just install the packages in the buildkite job that runs the RayDP test? I.e. move this to `.buildkite/pipeline.ml.yml` and then into the `Dataset tests ...` jobs.

To make this a bit nicer, maybe create a file `./ci/env/install-java.sh` where we just run the apt install command and refer to this in the buildkite jobs